.github/workflows/metadata.yml

```
@@ -3,30 +3,64 @@
+
+      - name: 🤖 Enable automerge for metadata PR
+        if: steps.cpr.outputs.pull-request-number
+        uses: peter-evans/enable-pull-request-automerge@v3
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          pull-request-number: ${{ steps.cpr.outputs.pull-request-number }}
+          merge-method: squash
+
+      - name: 📋 Show PR URL (debug)
+        if: steps.cpr.outputs.pull-request-url
+        run: echo "PR generado: ${{ steps.cpr.outputs.pull-request-url }}"
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 62: mapping values are not allowed in this context

(syntax-check)